app/models/item.rb

```diff
@@ -24,4 +24,15 @@ def self.top_by_number_sold(quantity = 5)
         order("sum(invoice_items.quantity) DESC").
         limit(quantity)
   end
+
+  def best_day
+    invoice = InvoiceItem.where(item_id: self.id).
+    select("invoice_items.quantity AS count").
+    joins(:invoices, :items).
+    group("invoices.id").
+    order("sum(count) DESC").
+    limit(1)
+    invoice.first.created_at
```

Comment:
i'm 90% sure I was moving in the wrong direction in this. Feeling rushed isn't conducive to good testing, and I wasn't able to build up the query successfully in the rails console, so... was mostly just shooting around in the dark.

Everything else about this method works - it returns the properly formatted date, it just... doesn't return the correct date.